app/guide/shared/Calculator/Calculator.tsx

```diff
@@ -176,33 +194,71 @@ export const Calculator = () => {
             events={events}
             setEvents={setEvents}
             lastWeekDay={lastWeekDay}
+            hasIncome={planType === "RS"}
           />
         ))}
         <div className="flex justify-end">
           <Button
             color="green"
             label="New sale"
             icon={PlusIcon}
-            onClick={() => setEvents([...events, getDefaultData(lastWeekDay)])}
+            onClick={() =>
+              setEvents([
+                ...events,
+                getDefaultData(lastWeekDay, !isPlanUsQualified),
+              ])
+            }
           />
         </div>
         <Section title="Summary">
-          <div className="grid gap-2">
+          <div className="grid grid-cols-1 gap-2">
+            {planType === "RS" ? (
+              <>
+                <div>
+                  <span>Income: </span>
+                  <Currency value={totals?.income ?? null} unit="eur" />
+                </div>
+                <div>
+                  <span>Income From French Origin: </span>
+                  <Currency value={totals?.incomeFr ?? null} unit="eur" />
+                </div>
+                <div className="border border-gray-300" />
+              </>
+            ) : null}
             <div>
               <span>Capital Gain (1133-A): </span>
-              <Currency value={capitalGain} unit="eur" />
+              <Currency value={totals?.gain ?? null} unit="eur" />
             </div>
             <div>
               <span>Capital Loss (1133-B): </span>
-              <Currency value={capitalLoss} unit="eur" />
+              <Currency value={totals?.loss ?? null} unit="eur" />
             </div>
             <div>
               <span>Capital Gain / Loss: </span>
               <Currency
-                value={capitalGain && capitalLoss && capitalGain + capitalLoss}
+                value={totals && totals.gain + totals.loss}
                 unit="eur"
               />
             </div>
+            {planType === "RS" ? (
+              <>
+                <div className="border border-gray-300" />
+                <div>
+                  <span>Taxable Income (1TZ): </span>
+                  <Currency
+                    value={totals ? (totals.incomeFr + totals.loss) / 2 : null}
```

Comment:
Not sure if applicable, but there is a cap of 300K, exceeding part should be reported in `1TT`

Reply:
That's a good point, I'll add that logic in